shell/renderer/renderer_client_base.cc

```diff
@@ -361,11 +361,20 @@ bool RendererClientBase::IsPluginHandledExternally(
 
 bool RendererClientBase::IsOriginIsolatedPepperPlugin(
     const base::FilePath& plugin_path) {
-#if BUILDFLAG(ENABLE_PDF_VIEWER)
-  return plugin_path.value() == kPdfPluginPath;
-#else
-  return false;
-#endif
+  // Taken from chrome's own implementation of this method.
```

Comment:
Can you link to it? Helpful if it changes in the future

Reply:
Changed to include a link to the method in question in the next commit 👍